### PR TITLE
dp test output files: energy per atom, virial per atom

### PR DESCRIPTION
**Pull request by likefallwind**
add two output files
https://github.com/deepmodeling/deepmd-kit/issues/2002

### CODE REVIEW

File: deepmd/entrypoints/test.py [hunk 1]
```diff
@@ -289,6 +289,12 @@ def test_ener(
             header="%s: data_e pred_e" % system,
             append=append_detail,
         )
+        pe_atom = pe / natoms
```
[the remark this answers is not in the document]
Comment (njzjz) [reply deepmd/entrypoints/test.py:292]: ... which are computed in line 252-253.

[the remark this answers is not in the document]
Comment (brucefan1983) [reply deepmd/entrypoints/test.py:292]: Just to clarity, what I asked for in the issue is the total energy for a structure divided by the number of atoms in the structure (that is, still only one energy for one structure), not the site energies on the idividual atoms.

[the remark this answers is not in the document]
Comment (brucefan1983) [reply deepmd/entrypoints/test.py:292]: For example, an output file for the test energy reads as follows:
```
# ../deepmd_data/hasvirial/C27: data_e pred_e
-4.153710174560546875e+01 -4.292935365276792936e+01
more lines
# ../deepmd_data/hasvirial/C4: data_e pred_e
-1.116660022735595703e+01 -1.022363460930032097e+01
more lines
```
Here I hope the energies for the C27 structures (27 carbon atoms) are divided by 27 and the energies for the C4 structures are divided by 4 such that I can get the "parity plot" straightforwardly (just plot the second column agains the first column).

[the remark this answers is not in the document]
Comment (njzjz) [reply deepmd/entrypoints/test.py:292]: Putting on the second column is a breaking change and may break the existing scripts. I would support appending to the last axis.

**Review by njzjz — APPROVED** [review]

**Pull request merged**